core/src/main/resources/i18n/displayStrings_ru.properties

```diff
@@ -669,16 +669,16 @@ portfolio.pending.autoConf.state.ERROR=An error at a service request occurred. N
 # suppress inspection "UnusedProperty"
 portfolio.pending.autoConf.state.FAILED=A service returned with a failure. No auto-confirm possible.
 
-portfolio.pending.step1.info=Deposit transaction has been published.\n{0} must wait for at least one blockchain confirmation before starting the payment.
-portfolio.pending.step1.warn=The deposit transaction is still not confirmed. This sometimes happens in rare cases when the funding fee of one trader from an external wallet was too low.
-portfolio.pending.step1.openForDispute=The deposit transaction is still not confirmed. You can wait longer or contact your mediator for assistance.
+portfolio.pending.step1.info=Транзакция депозита опубликована.\n{0} необходимо дождаться как минимум одного подтверждения блокчейна, прежде чем начать платеж.
+portfolio.pending.step1.warn=Депозитная транзакция все ещё не подтверждена. Это иногда случается в редких случаях, когда комиссия за финансирование одного трейдера с внешнего кошелька была слишком низкой.
+portfolio.pending.step1.openForDispute=Депозитная транзакция все ещё не подтверждена. Вы можете подождать дольше или обратиться за помощью к своему посреднику.
 
 # suppress inspection "TrailingSpacesInProperty"
-portfolio.pending.step2.confReached=Your trade has reached at least one blockchain confirmation.\n\n
+portfolio.pending.step2.confReached=Ваша сделка получила как минимум одно подтверждение в блокчейне.\n\n
 
-portfolio.pending.step2_buyer.refTextWarn=Important: when making the payment, leave the \"reason for payment\" field empty. DO NOT put the trade ID or any other text like 'bitcoin', 'BTC', or 'Bisq'. If you are required to fill in a reason, use your account name eg "Joe Bloggs", alternatively enter something non-descriptive like a dash (-). You are free to discuss via trader chat if an alternate \"reason for payment\" would be suitable to you both.
+portfolio.pending.step2_buyer.refTextWarn=Важно: при совершении платежа оставьте поле \«причина платежа\» пустым. НЕ вводите идентификатор сделки или любой другой текст, например ''биткоин'', ''bitcoin'', ''BTC'' или ''Bisq''. Если вам необходимо указать причину, используйте имя своей учетной записи, например, \«Джон Блоггс\», или введите что-то неописательное, например, тире (-). Вы можете свободно обсудить в чате трейдеров, подойдет ли вам обоим альтернативная \«причина платежа\».
```

Comment:
As here is not variable used (e.g. {0}) the duplicate single quote is not needed and would get displayed as duplicate quote. Only if a variable and a single quote occurs in a string we need to duplicate it.

Reply:
Good catch, I fixed that one plus all the other incorrect doubling up's (or missing doubling up's) in the default (English) and other language files.  However, French and Italian have a large number of incorrect apostrophes because of stress marks used in the language itself, so fixing that will need to be a separate pull request because it's far outside the scope of this pull request adding Russian translations.